src/transactions/BumpSequenceTests.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "crypto/SignerKey.h"
+#include "lib/catch.hpp"
+#include "main/Application.h"
+#include "main/Config.h"
+#include "overlay/LoopbackPeer.h"
+#include "test/TestAccount.h"
+#include "test/TestExceptions.h"
+#include "test/TestUtils.h"
+#include "test/TxTests.h"
+#include "test/test.h"
+#include "transactions/TransactionFrame.h"
+#include "util/Logging.h"
+#include "util/Timer.h"
+#include "util/make_unique.h"
+
+using namespace stellar;
+using namespace stellar::txtest;
+
+typedef std::unique_ptr<Application> appPtr;
+
+template <typename Callable>
+void
+for_versions_bumpseq(ApplicationEditableVersion& app, Callable&& f) {
+    for_versions_to(std::max(8, //Config::CURRENT_LEDGER_PROTOCOL_VERSION // TODO: For some reason I'm getting a linker error on this const?
```

Comment:
I think it should be for_versions_from(9, ) here
and that it needs a counterpart: for_versions_bumpseq with for_versions_to(8) that expects NOT_SUPPORTED_YET everytime